pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/GivenNameDAOImpl.java

```diff
@@ -0,0 +1,102 @@
+package gov.ca.emsa.pulse.broker.dao.impl;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Query;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
```

Comment:
I was also looking at all of the domain objects down a few lines. Not sure if they're required too...